proposals/cleanup.md

```diff
@@ -46,15 +46,15 @@ There are two main use cases associated with this.
 
 In this proposal, Kyverno may function as a clean-up controller allowing it to identify and scavenge resources. There are two proposed capabilities which would be used to effect a clean-up.
 
-1. Use of two specific annotations which may be set on any resource:
-   - `kyverno.io/ttl`: Sets the time-to-live a resource may have which functions as a count-down timer starting from the time at which the annotated resource was either created or when the annotation was assigned. Value is a string and units are given in either minutes, hours, or days. Ex., `kyverno.io/ttl: 30m`, `kyverno.io/ttl: 30d`
+1. Use of two specific labels which may be set on any resource:
+   - `kyverno.io/ttl`: Sets the time-to-live a resource may have which functions as a count-down timer starting from the time at which the labelled resource was either created or when the label was assigned. Value is a string and units are given in either minutes, hours, or days. Ex., `kyverno.io/ttl: 30m`, `kyverno.io/ttl: 30d`
```

Comment:
I would prefer something like `cleanup.kyverno.io/ttl`

Reply:
> or when the label was assigned

I don't think we can do that, it should always be relative to the resource creation time.

Reply:
done